src/vs/workbench/api/common/extHostTypeConverters.ts

```diff
@@ -1433,6 +1433,18 @@ export namespace NotebookCellMetadata {
 	}
 }
 
+export namespace NotebookCellInternalMetadata {
```

Comment:
There already exists `NotebookCellExecutionSummary.from` and we should merge the `to` util into that